src/components/calendar/ClassModal.js

```diff
@@ -26,6 +28,21 @@ function ClassModal({ showDialog, toggleDialog, classes, section, removeSection
         <Typography variant="h5">
           {`${section.subjectId} ${section.courseId} Section ${section.sectionNumber}`}
         </Typography>
+        <Typography>
+          {`${section.schedules.map(getFormattedClassSchedule)}`}
+        </Typography>
+        <Typography>
+          {`${section.schedules.map(schedule => schedule.location)}`}
+        </Typography>
+        <Typography gutterBottom="true">
```

Comment:
So two things here. First, you probably meant to provide the JS boolean value `true` instead of the string `"true"`. Second, when you want a prop to have the value of `true`, you actually never write the value in JSX. It's a shorthand, and linting will enforce it. So it should just be `<Typography gutterBottom>`.